File: BackEndAPI/Data/Entities/DonHang.cs
using BackEndAPI.Data.Enums;
using System;
using System.Collections.Generic;

#nullable disable

namespace BackEndAPI.Entities
{
    public partial class DonHang
    {
        public int MaDonHang { get; set; }
        public int MaNguoiDung { get; set; }
        public int MaCuaHang { get; set; }
        public int? MaShipper { get; set; }
        public string DiaChi { get; set; }
        public DateTime? NgayMua { get; set; }
        public TrangThaiDonHang TrangThai { get; set; }
        public DateTime? NgayCapNhat { get; set; }
        public string TenNguoiNhan { get; set; }
        public string Sdt { get; set; }
        public string PhanHoi { get; set; }

        public virtual CuaHang CuaHang { get; set; }
        public virtual NguoiDung NguoiMua { get; set; }
        public virtual NguoiDung Shipper { get; set; }
        public virtual ICollection<ChiTietDonHang> DSChiTietDonHang { get; set; }
    }
}

File: BackEndAPI.Java/DiChoThue/src/main/java/com/example/DiChoThue/Entities/DonHang.java
package com.example.DiChoThue.Entities;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.util.Date;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "DON_HANG")
public class DonHang {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int ma_don_hang;
    private int ma_nguoi_dung;
    private int ma_cua_hang;
    private int ma_shipper;
    private String dia_chi;
    private Date ngay_mua;
    private int trang_thai;
    private Date ngay_cap_nhat;
    private String ten_nguoi_nhan;
    private String sdt;
    private String phan_hoi;
    private int tongtien;
}
